lib/irc/ConnectionInstance.js

```diff
@@ -355,9 +355,16 @@ ConnectionInstance.create = Promise.coroutine(function*(server, opts, onCreatedC
             log.error(
                 `ConnectionInstance.connect failed after ${connAttempts} attempts (${err.message})`
             );
+
+            if (err.message === "net_error") {
```

Comment:
This is a strange way of backing off. Should this be an `if (... && err.message === "net_error")` with the if statement below?